src/style.css

```
@@ -1 +1,29 @@
 /* styles go here */
+
+.form-field {
+  margin-bottom: 10px;
+}
+
+.form-field:first-child {
+  margin-top: 0;
+}
```

you have the same style for .information:first-child so not needed to create separated style block
you can do so
```suggestion
.form-field:first-child, .information:first-child {
  margin-top: 0;
}
```